system/t06_publish/AzurePublish2Test_gold

```diff
@@ -3,6 +3,7 @@ Generating metadata files and linking package files...
 Finalizing metadata files...
 Signing file 'Release' with gpg, please enter your passphrase when prompted:
 Clearsigning file 'Release' with gpg, please enter your passphrase when prompted:
-Cleaning up prefix "." components main...
+Cleaning up published repository azure:test1:./maverick...
```

Comment:
there is a mix of quoted, double quotes, back quoted and not quoted repo/mirror/component references... should we clean that up ?

Reply:
I will correct that by using `'` instead of `"`.

Reply:
It would be nice to quote the `storage:prefix/distribution` as well, but that requires a lot of modifications throughout the whole code. Maybe we can do that when this PR has been merged.